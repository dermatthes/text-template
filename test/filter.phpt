<?php
namespace Leuffen\TextTemplate;

require __DIR__ . "/../vendor/autoload.php";


use Tester\Assert;



\Tester\Environment::setup();


$p = new TextTemplate(file_get_contents(__DIR__ . "/unit/filter.in.txt"));
Assert::equal(
    file_get_contents(__DIR__ . "/unit/filter.expected.txt"),
    $p->apply(["str" => "abc123'\"", "strMultiLine" => "a\nb\nc"])
    , "Error in filter-test"
);





