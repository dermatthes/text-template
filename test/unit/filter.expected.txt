#1 default      : abc123'&quot;
#2 raw          : abc123'"
#3 singleLine   : a b c
#4 inivalue     : abc123\'\&quot;
#5 html         : abc123'&amp;quot;
#6a fixedLength : ab
#6b fixedLength : abc123'&quot;++++++++++++++++++++++